collectors/cgroups.plugin/sys_fs_cgroup.c

```diff
@@ -160,7 +160,18 @@ static enum cgroups_type cgroups_try_detect_version()
     if(!cgroups2_available)
         return CGROUPS_V1;
 
-    // 2. check systemd compiletime setting
+#if defined CGROUP2_SUPER_MAGIC
+    // 2. check filesystem type for the default mountpoint
+    struct statfs fsinfo;
+    if (!statfs("/sys/fs/cgroup", &fsinfo)) {
```

Comment:
> !statfs("/sys/fs/cgroup")

Should we use `netdata_host_prefix` here?

Reply:
Ups :grin:! Thank you. Fixed :innocent: